data/hai/hai reveal 1 intro.txt

```diff
@@ -309,15 +307,66 @@ mission "ZZ Event Hai Secret Leaks: FW"
 		event "news: hai video clip published"
 		event "pirates investigate leaks"
 		conversation "hai reveal free worlds news"
-	
-	
+
+
 event "pirates investigate leaks"
 	system "Waypoint"
 		government "Pirate"
 		add fleet "Marauder fleet IX" 2000
 		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700
 
 
+mission "Trader Directions to Hai Space"
+	landing
+	name "Investigate the Hai"
+	description "Follow the coordinates given to you by Captain Casey and meet the Hai."
+	source Sunracer
+	waypoint "Ultima Thule"
+	destination "Allhome"
+	to offer
+		or
+			has "ZZ Event Hai Secret Leaks: RS: offered"
+			has "ZZ Event Hai Secret Leaks: FW offered"
+	on offer
+		conversation
+			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that here, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here on Sunracer, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`
+			`	The endless cycle of both the news and government propaganda being displayed on most screens in major thoroughfares is remarkably absent here, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`
+			`	Casting around the room you pick out a merchant who looks like they've been around the block a few times. In addition to being a woman who looks like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. As you make your way over though you see her glare falter in a flash of recognition, before subsiding to a reserved scowl.`
+			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention hey?"`
```

Comment:
```suggestion
			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention today?"`
```

Reply:
I was under the impression that this was a typo, is there something I'm missing here?

Reply:
From Wiki -> "Eh" can also be added to the end of a declarative sentence to turn it into a question seeking agreement. For example: "The weather is nice." becomes "The weather is nice, eh?". This same phrase could also be taken as "The weather is nice, don't you agree?". In this usage, it is similar to [Scots](https://en.wikipedia.org/wiki/Scots_language) "eh?", [English](https://en.wikipedia.org/wiki/English_language) "innit?", [Portuguese](https://en.wikipedia.org/wiki/Portuguese_language) "né?", [Dutch](https://en.wikipedia.org/wiki/Dutch_language) "hè?", [Japanese](https://en.wikipedia.org/wiki/Japanese_language) "ne?", [Mandarin](https://en.wikipedia.org/wiki/Standard_Chinese) "bā" or [French](https://en.wikipedia.org/wiki/French_language) "hein?" (which differs from the French usage of "n'est-ce pas?" ("Is it not?") in that it does not use a (technically double or emphatic) negative).

The equivalent in [South African English](https://en.wikipedia.org/wiki/South_African_English) and most of Southern and Western Australia is "hey".

In oceanic dialects of English (Aus, NZ) both "eh" and "hey" can further be added to the end of an actual question, which is typically otherwise ridiculous to ask, to indicate that the question is rhetorical, subsequently re-inferring the content of the question as a declarative statement. 

In this case it infers the direct statement, "I must have offended an ancient Sumerian deity to be receiving the misfortune of your attention."
-------------------------------------

Until roughly 20 minutes ago I did not realise that this phraseology was as regionally unique as it turns out to be...

Reply:
Huh, I had no idea. My bad. :P